src/Model.php

```diff
@@ -912,12 +1173,18 @@ public function join($foreign_table, $defaults = [])
         return $this->add(new $c($defaults));
     }
 
+    /**
+     * Left Join support.
+     *
+     * @see join()
+     *
+     * @param string $foreign_table
+     * @param array  $defaults
+     *
+     * @return Join
+     */
     public function leftJoin($foreign_table, $defaults = [])
     {
-        if (!is_array($defaults)) {
-            $defaults = ['master_field' => $defaults];
-        }
-
```

Comment:
Yes, that's my mistake.
It looked that `join()` have same checking so I removed this "duplicated" one, but I didn't notice, that `$defaults['weak'] = true;` line which changes situation :(